packages/esm-patient-chart-app/src/banner/patient-banner.component.tsx

```diff
@@ -118,8 +122,8 @@ const PatientBanner: React.FC<PatientBannerProps> = ({
           </div>
           <div className={styles.row}>
             <div className={styles.identifiers}>
-              {patient?.identifier?.length
-                ? patient?.identifier.map(({ value, type }) => (
+              {filterPatientIdentifier?.length
+                ? filterPatientIdentifier.map(({ value, type }) => (
```

Comment:
I'd change this to `filterPatientIdentifier?.map()` - no need for ternary operator as `null` or `undefined` will not render. If the length is zero you get an empty array, which also will not render. So you wrote less code that is actually also better for React (right now we still have one text node in case nothing should be rendered).